lisa/sut_orchestrator/azure/common.py

```diff
@@ -26,6 +26,7 @@
 
 AZURE = "azure"
 AZURE_SHARED_RG_NAME = "lisa_shared_resource"
+AZURE_SHARED_RG_LOCATION = "westus2"
```

Comment:
use the `RESOURCE_GROUP_LOCATION`. They are the same purpose.

Reply:
Does this mean that resource groups which are created by LISA for hosting VM's always in "westus2"?

Reply:
It's used for resource group only, it doesn't impact where VM is deployed.